doc/source/whatsnew/v0.24.0.rst

```diff
@@ -280,6 +280,7 @@ Other Enhancements
 - :meth:`round`, :meth:`ceil`, and meth:`floor` for :class:`DatetimeIndex` and :class:`Timestamp` now support a ``nonexistent`` argument for handling datetimes that are rounded to nonexistent times. See :ref:`timeseries.timezone_nonexistent` (:issue:`22647`)
 - :class:`Resampler` now is iterable like :class:`GroupBy` (:issue:`15314`).
 - :meth:`Series.resample` and :meth:`DataFrame.resample` have gained the :meth:`Resampler.quantile` (:issue:`15023`).
+- :class:`PeriodIndexResampler` aggregations will now respect the base argument in the same fashion as :class:`DatetimeIndexResampler` (:issue:`23882`)
```

Comment:
can you put double backticks around base

Reply:
done